main.go

```diff
@@ -51,8 +52,34 @@ func startAMQChecks() {
 	}
 }
 
+func startSSOChecks() {
+	url := "https://keycloak-edge-redhat-rhmi-rhsso.apps.sso-128.y3z4.s1.devshift.org"
+	user := "admin"
+	pwd := "NH0trX4G93snKw=="
+	realm := "master"
+	if url != "" && user != "" && pwd != "" && realm != "" {
+		log.WithFields(log.Fields{
+			"serverURL": url,
+			"user":      user,
+			"password":  pwd,
```

Comment:
be careful not to log sensitive data in logs.

Reply:
Sure. Will remove the sensitive data from logs